* d/p/load-versioned-libcc1.patch:
  - load libcc1.so.0 instead unversioned file.
Author: Hector Oron <zumbi@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: Debian
Forwarded: not-needed
Last-Update: <2015-03-23>

--- gdb-7.9.orig/include/gcc-c-interface.h
+++ gdb-7.9/include/gcc-c-interface.h
@@ -197,7 +197,7 @@ struct gcc_c_context
 /* The name of the .so that the compiler builds.  We dlopen this
    later.  */
 
-#define GCC_C_FE_LIBCC libcc1.so
+#define GCC_C_FE_LIBCC libcc1.so.0
 
 /* The compiler exports a single initialization function.  This macro
    holds its name as a symbol.  */
